CHANGELOG.md

```diff
@@ -1,8 +1,22 @@
-# dbt_apple_search_ads version.version
+# dbt_apple_search_ads v0.5.0
+[PR #29](https://github.com/fivetran/dbt_apple_search_ads/pull/29) includes the following updates:
+
+## Breaking Changes: Apple Search Ads API v5 schema updates
+- Following the connector updates surrounding the Apple Search Ads API v5 release, the following fields have been added to the upstream `*_report` staging models and will be used to sunset the below fields:
+  - `tap_installs` - replacing `conversions`
+  - `tap_new_downloads` - replacing `new_downloads`
+  - `tap_redownloads` - replacing `redownloads`
```

Comment:
Again, I know i'm not supposed to look at the changelog rn, but I think it could be good to mention that we are coalescing the new fields with the old to ensure historical data is persisted

Reply:
yes updated this! thanks!